<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmbeddedDesignerForm</class>
 <widget class="QWidget" name="EmbeddedDesignerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>530</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="editWidgetsBtn">
        <property name="text">
         <string>Edit Widgets</string>
        </property>
        <property name="icon">
         <iconset resource="EmbeddedDesigner.qrc">
          <normaloff>:/images/images/widgettool.png</normaloff>:/images/images/widgettool.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="id" stdset="0">
         <string>widgets</string>
        </property>
        <attribute name="buttonGroup">
         <string>editSwitch</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="editTabBtn">
        <property name="text">
         <string>Edit Tab Order</string>
        </property>
        <property name="icon">
         <iconset resource="EmbeddedDesigner.qrc">
          <normaloff>:/images/images/tabordertool.png</normaloff>:/images/images/tabordertool.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="id" stdset="0">
         <string>tabs</string>
        </property>
        <attribute name="buttonGroup">
         <string>editSwitch</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="editBuddyBtn">
        <property name="text">
         <string>Edit Buddies</string>
        </property>
        <property name="icon">
         <iconset resource="EmbeddedDesigner.qrc">
          <normaloff>:/images/images/buddytool.png</normaloff>:/images/images/buddytool.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="id" stdset="0">
         <string>buddies</string>
        </property>
        <attribute name="buttonGroup">
         <string>editSwitch</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widgetsPlace" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QMdiArea" name="formPlace">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>44</red>
         <green>42</green>
         <blue>42</blue>
        </color>
       </brush>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="objectsPlace" native="true">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="propertyPlace" native="true">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="EmbeddedDesigner.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="editSwitch"/>
 </buttongroups>
</ui>
